SPECS/python3/python3.spec

```
@@ -3,32 +3,25 @@
 # Automating the extraction of these alternate version strings has proven to be tricky,
 # with regards to tooling available in the toolchain build environment.
 # These will be manually maintained for the time being.
-%global majmin 3.9
-%global majmin_nodots 39
-# See Lib/ensurepip/__init__.py in Source0 for these version numbers
-%global pip_version 22.0.4
-%global setuptools_version 58.1.0
+%global majmin 3.12
+%global majmin_nodots 312
```

Similar small suggestion to reduce manual changes:
```suggestion
%define majmin_nodots %(echo %{majmin} | tr -d .)
```

---

thanks, will fix